extristric:
- - -0.015322395778833922
  - 0.01971269817572972
  - 0.17993504091676385
- - 0.10192273132740215
  - 0.9617233690785127
  - -0.2543617860588287
- - 0.9884638877345611
  - -0.12670326378599595
  - -0.08297846461976584
- - -0.11203079703117223
  - -0.24297004818341572
  - -0.9635427630376915
hand2cam_transformation_matrix:
- - -0.01782217674806108
  - -0.1046008609006226
  - 0.9943545795714974
  - -0.09630850772104702
- - -0.9998308045282818
  - 0.006393502312188604
  - -0.01724776636300579
  - -0.009750665301640829
- - -0.004553277093442498
  - -0.9944937320199825
  - -0.10469710903686735
  - 0.07899833508544073
- - 0.0
  - 0.0
  - 0.0
  - 1.0
white_area_pose_cam:
- - 0.10236127136228967
  - -0.9637713095308662
  - 0.2462962171798047
  - -0.015332318152416502
- - 0.9889881408305115
  - 0.125202306259356
  - 0.07889538467342798
  - 0.019739976155714607
- - -0.10687471265354376
  - 0.2355081048617303
  - 0.9659744412712532
  - 0.18014933117707782
- - 0.0
  - 0.0
  - 0.0
  - 1.0
white_area_pose_world:
- - -0.9872629227315989
  - -0.06811584330188533
  - 0.14377022211683335
  - 0.1742838995346352
- - -0.10236127136228952
  - 0.9637713095308662
  - -0.24629621717980488
  - 0.026332318152416486
- - -0.12178533699764581
  - -0.25787577792320543
  - -0.9584681664742728
  - 0.02508812273972358
- - 0.0
  - 0.0
  - 0.0
  - 1.0
